ludwig/features/audio_feature.py

```diff
@@ -469,21 +457,23 @@ def forward(self, inputs, mask=None):
 
     @property
     def input_shape(self) -> torch.Size:
-        return torch.Size([self.max_sequence_length, self.embedding_size])
+        return torch.Size([self.encoder_config.max_sequence_length, self.encoder_config.embedding_size])
 
     @property
     def input_dtype(self):
         return torch.float32
 
     @staticmethod
     def update_config_with_metadata(input_feature, feature_metadata, *args, **kwargs):
-        input_feature["max_sequence_length"] = feature_metadata["max_length"]
-        input_feature["embedding_size"] = feature_metadata["feature_dim"]
-        input_feature["should_embed"] = False
+        input_feature[ENCODER]["max_sequence_length"] = feature_metadata["max_length"]
+        input_feature[ENCODER]["embedding_size"] = feature_metadata["feature_dim"]
+        input_feature[ENCODER]["should_embed"] = False
 
     @staticmethod
     def populate_defaults(input_feature):
-        set_default_values(input_feature, {TIED: None, "preprocessing": {}})
+        defaults = AudioInputFeatureConfig()
+        set_default_values(input_feature, {TIED: defaults.tied, PREPROCESSING: {}})
+        set_default_values(input_feature, {ENCODER: {TYPE: defaults.encoder.type}})
```

Comment:
nit: This could be one statement.

```python
set_default_values(
    input_feature, 
    {
        TIED: defaults.tied, 
        PREPROCESSING: {}, 
        ENCODER: {TYPE: defaults.encoder.type}
    }
)
```